android/java/org/chromium/chrome/browser/BraveActivity.java

```diff
@@ -92,12 +92,6 @@ public void onStartWithNative() {
 
         // Disable page prefetching
         PrefetchConfiguration.setPrefetchingEnabledInSettings(false);
-
-        // Set a flag in preferences that tabs pref migration is done
-        SharedPreferences.Editor sharedPreferencesEditor =
-            ContextUtils.getAppSharedPreferences().edit();
-        sharedPreferencesEditor.putBoolean(BraveHelper.PREF_TABS_SETTINGS_MIGRATED, true);
-        sharedPreferencesEditor.apply();
```

Comment:
There was a race condition that Activity starts faster than our Upgrade service done it's work. I decided that it's just safe to check on 0 for blocked items as they will be 0 for new version in any case and we set them to 0 after migration is done.